packages/measure/src/measure-renders.tsx

```diff
@@ -69,14 +81,19 @@ export async function measureRender(ui: React.ReactElement, options?: MeasureOpt
     runResults.push({ duration, count });
   }
 
+  if (hasUnnecessaryRender) {
+    const testName = expect.getState().currentTestName;
+    logger.warn(`test "${testName}" has unnecessary renders. Please update your code to avoid unnecessary renders.`);
```

Comment:
```suggestion
    logger.warn(`test "${testName}" has ${initialRendersCount - 1} unnecessary initial render(s). Please update your code to avoid unnecessary renders.`);
```